manifests/zone.pp

```diff
@@ -14,46 +15,59 @@
     $allow_transfer = [],
     $allow_query    = [],
     $also_notify    = [],
-    $zone           = $title,
-    $contact        = "root.${title}.",
+    $zone           = undef,
+    $contact        = undef,
     $zonefilepath   = $::dns::zonefilepath,
-    $filename       = "db.${title}",
+    $filename       = undef,
```

Comment:
I think you are right. One does not want a shared zonefile for a zone in multiple views anyway. Bind will rename such a zonefile to something like db-xyzabcd.